app/Note.php

```diff
@@ -4,35 +4,60 @@
 
 use App\Helpers\DateHelper;
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Parsedown;
 
+/**
+ * @property Account $account
+ * @property Contact $contact
+ * @property string $parsed_body
+ */
 class Note extends Model
 {
     /**
+     * The attributes that aren't mass assignable.
+     *
+     * @var array
+     */
+    protected $guarded = ['id'];
+
+    /**
      * Get the account record associated with the note.
+     *
+     * @return BelongsTo
      */
     public function account()
     {
-        return $this->belongsTo('App\Account');
+        return $this->belongsTo(Account::class);
     }
 
     /**
      * Get the contact record associated with the note.
+     *
+     * @return BelongsTo
      */
     public function contact()
     {
-        return $this->belongsTo('App\Contact');
+        return $this->belongsTo(Contact::class);
+    }
+
+    /**
+     * Return the markdown parsed body
+     *
+     * @return string
+     */
+    public function getParsedBodyAttribute()
```

Comment:
I thought it might be a good addition, but only parsing is possible right now and it's not used anywhere in this PR.